docs/source/how-tos/To-do-list-for-New-MOC-Intern-and-Member.md

```diff
@@ -82,30 +120,20 @@ You can expect at most a couple of questions in a day, and likely less.
  If you get more than that, then contact admin@lists.massopen.cloud and we will get you more help.
```

Comment:
We hope you get more than that (and please thank us for it), but also contact us if you need more help :).  I think this is a future edit.

Reply:
@msdisme will rewrite the whole MOC community section, so I am gonna leave it as it is for now.